testng-core/src/test/java/test/retryAnalyzer/github2669/IssueTest.java

```diff
@@ -0,0 +1,19 @@
+package test.retryAnalyzer.github2669;
+
+import org.testng.Assert;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+import java.util.Collections;
+
+public class IssueTest extends SimpleBaseTest {
```

Comment:
Can you please remove this test class and instead add this test method (with a description that refers to the github issue) inside `test.retryAnalyzer.RetryAnalyzerTest` ? We would like to get away from this practise of having all these `IssueTest` but instead have one class that has tests for everything.